test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -407,7 +413,9 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 			ImportTest::compareStates(postState, blockchain.topBlock().state());
 		}
 	}//for tests
-	TestOutputHelper::finishTest();
+
+	if (!skipTestOutput)
+		TestOutputHelper::finishTest();
```

Comment:
it is still being executed when fillchain == false. 
it should be executed only in blockchain tests and not in bcRandom